Solution1/Solution1.Tests/BL/Services/WeatherServiceTests.cs

```diff
@@ -42,6 +54,58 @@ public async Task GetByCityNameAsync_ReturnedWeatherDTO_Success()
             // Assert
             var expectedWeatherDto = new WeatherDTO() { CityName = cityName, Temp = temp, Comment = "It's fresh." };
             Assert.True(new CompareLogic().Compare(expectedWeatherDto, result).AreEqual);
-        }               
+        }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(3)]
+        public async Task GetForecastByCityNameAsync_ReturnedForecastWeatherDTO_Success(int countDay)
+        {
+            // Arrange
+            var cityName = "Minsk";
+            var countWeatherPoint = 8*countDay + (DateTime.UtcNow.Date.AddDays(1) - DateTime.UtcNow).Hours / 3;
+            var forecastWeatherApiDTO = new ForecastWeatherApiDTO()
+            {
+                City = new CityApiDTO() { Name = cityName },
+                WeatherPoints = new List<WeatherInfoApiDTO>()
+            };
+
+            for(int i = 1; i <= countDay; i ++)
```

Comment:
give meaningful names to indexes, and get rid of magic numbers

Reply:
+